CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ We use [semantic versioning](http://semver.org/):
 
 # Next Release
 - [feature] Option `teamscale-revision-manifest-jar` which can be used to read the git commit revision from the `META-INF/MANIFEST.MF` in the Jar file. The field must be named `Revision` and must be provided as a main attribute. Alternatively the field can be called `Git_Commit` and must be an attribute of an entry called `Git`.
+- [fix] _teamscale-client_: SSL validation is now enabled by default
```

Comment:
This should be `[breaking change]` as it cannot be release as a patch update and also does not only hold for teamscale client but also for the jacoco agent and the teamscale gradle plugin